Platforms/Export-Import-Platform.ps1

```diff
@@ -178,4 +208,4 @@ else
     Write-Error "This script requires PowerShell version 3 or above"
 }
 
-Write-Host "Export / Import Platform: Script Started" -ForegroundColor Cyan
\ No newline at end of file
+Write-Host "Export / Import Platform: Script Started" -ForegroundColor Cyan
```

Comment:
Just noticed this, this is the last line of the script so needs to be changed from Started to Ended... :)

Reply:
Fixed